### DISCUSSION TITLE
官网中文文档翻译释义错误

### DISCUSSION THREAD

**Discussion by Dazhuwei** [apache/doris#16964] [RFC]
![image](https://user-images.githubusercontent.com/77066234/220233695-6dcd9adc-d8fc-4987-a14d-acd4876a42b2.png)
![image](https://user-images.githubusercontent.com/77066234/220233729-7d8e0273-fd49-44fe-b3d5-88e013dbe782.png)
https://doris.apache.org/zh-CN/docs/dev/data-operate/export/export-manual/
数据操作-->导出-->数据导出
导出到对象存储-->通过s3 协议直接将数据导出到指定的存储.
中文文档中AWS_ENDPOINT和AWS_REGION 中文翻译释义颠倒了，英文文档正常。希望改正下。

**Comment by Dazhuwei** [comment]
提了Pr 
[fix](doc) fix export-manual.md #16969

**Reply by zy-kkk** [reply]
Thank you for your service to the Doris community
